src/views/LoginPage/index.js

```diff
@@ -16,17 +24,44 @@ class LoginPage extends Component {
   }
 
   handleChange = event => {
-    const { info, value } = event.target;
-
-    this.setState({
-      [info]: value
-    });
-    console.log(this.state);
+    this.setState({ [event.target.name]: event.target.value });
   };
 
   handleSubmit(event) {
     event.preventDefault();
-    console.log(this.state);
+    var attributeList = [];
+    var dataEmail = {
+      Name: "email",
+      Value: this.state.email
+    };
+    var dataPassword = {
+      Name: "password",
+      Value: this.state.password
+    };
+    attributeList.push(dataEmail);
+    attributeList.push(dataPassword);
+
+    var authenticationDetails = new AmazonCognitoIdentity.AuthenticationDetails(
+      attributeList
+    );
+
+    var userData = {
+      Username: this.state.email,
+      Pool: UserPool
+    };
+
+    var cognitoUser = new AmazonCognitoIdentity.CognitoUser(userData);
+
+    cognitoUser.authenticateUser(authenticationDetails, {
+      onSuccess: function(result) {
+        var accessToken = result.getAccessToken().getJwtToken();
+        console.log(accessToken);
```

Comment:
remove this console.log and replace it with storing the access token in localStorage like [this](https://www.robinwieruch.de/local-storage-react)